### PR TITLE
Feat/45/complete registration of commander

### PR DESCRIPTION
**Pull request by mershalova**

### CODE REVIEW

**Review by 5haki6 — APPROVED** [review]

**Review by 5haki6 — COMMENTED** [review]
add to complete registration foot size, head size, gas mask size, blood type and uniform type

File: MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+    private _attributeService: AttributeService,
+    private messageService: MessageService,
+    private _router: Router) { }
+
+  ngOnInit(): void {
+    this._divisionsService.GetAllDivisions().subscribe((divisions)=>{this.divisions = divisions});
+    this._positionService.collection.getAll().subscribe((positions)=>{this.positions = positions});
+    this._rankService.collection.getAll().subscribe((ranks)=>{this.ranks = ranks});
+    this._attributeService.collection.getAll().subscribe((attributes)=>{this.attributes = attributes});
+    this.blood_types = ['I', 'II', 'III', 'IV'];
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts]: Why don't you get these data from the DB?

File: MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+import { ProfileModel } from 'src/app/shared/models/profile.model';
+
+@Component({
+  selector: 'app-finish-registration',
+  templateUrl: './finish-registration.component.html',
+  styleUrls: ['./finish-registration.component.scss']
+})
+export class FinishRegistrationComponent implements OnInit {
+
+  useRedClass: boolean = false;
```
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts:26]: Investigate the [reactive forms](https://angular.io/guide/reactive-forms)

File: MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts (hunk 1 above)
Comment (vladtymo) [inline MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts]: Seed the all possible data for these fields.

**Review by vladtymo — CHANGES_REQUESTED** [review]

**Review by Alex-knu — APPROVED** [review]

File: MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts (hunk 1 above)
Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts]: We do

Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts]: From now

Comment (Zhykovskiy) [reply MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts]: I will do it on branch 66

File: MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts (hunk 2 above)
Comment (mershalova) [reply MilitaryManager/client-app/src/app/components/finish-registration/finish-registration.component.ts:26]: added reactive forms

**Review by vladtymo — APPROVED** [review]

**Pull request merged**